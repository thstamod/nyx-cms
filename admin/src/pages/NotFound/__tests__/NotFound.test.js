import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import NotFound from '../NotFound';

test('404 page Page renders', () => {
  const { asFragment } = render(<NotFound />);

  expect(asFragment()).toMatchSnapshot();
});
