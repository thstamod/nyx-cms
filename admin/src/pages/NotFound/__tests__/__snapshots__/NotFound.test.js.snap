// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page Page renders 1`] = `
<DocumentFragment>
  <h1>
    404 not found
  </h1>
</DocumentFragment>
`;
